Framework/Muon/src/ConvertFitFunctionForMuonTFAsymmetry.cpp

```diff
@@ -46,8 +46,7 @@ std::string rmInsertFunction(const std::string &originalTie) {
   return LHName + RHName;
 }
 
-template <typename T1>
-int findName(const std::vector<std::string> &colNames, const T1 &name) {
+int findName(const std::vector<std::string> &colNames, std::string &name) {
```

Comment:
In this case you could just go fo `findName(const std::vector<std::string> &colNames, std::string name)` and take the string by value. When you call it you can do
```
auto wsNamesIndex = findName(colNames, "name");
```
and the `std::string` will move constructed as a `std::string` is able to be constructed from a `const char *`.